python/ray/serve/_private/utils.py

```diff
@@ -14,7 +14,6 @@
 from functools import wraps
 from typing import Any, Callable, Dict, List, Optional, TypeVar, Union
 
-import __main__
```

Comment:
what is this? (my first time seeing this...)

Reply:
I wrote it a long time ago but have no idea what it's for now lol